survey-thank-you.html

```diff
@@ -0,0 +1,81 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <meta name="description" content="Ally seamlessly integrates with the Learning Management System and automatically generates alternative accessible formats, provides instructor feedback and generates an institutional accessibility report">
+    <meta name="author" content="Fronteer LTD">
+    <link rel="icon" href="favicon.ico">
+
+    <title>Ally - Making course content accessible</title>
```

Comment:
The title should describe the page. Let's make this something along the lines of "Ally - User Group Survey"